Cargo.toml

```diff
@@ -25,7 +25,7 @@ mio-extras = "2.0"
 rand = "0.6"
 sha-1 = "0.8.0"
 slab = "0.4"
-url = "1.7.0"
+url = "2.0.0"
```

Comment:
What about allowing the 1.7 version as well?  
Something like
```suggestion
url = ">=1.7, <3.0.0"
```

Reply:
@nox, @Eijebong I vaguely recall that using this pattern for `serde` in `url` had downsides, but I don’t remember what they were. What do you think?

Reply:
It's a bad idea, it makes it look like end users can continue to use 1.7, but cargo will happily bring url 2 in their project anyway and then you have "expected `Url`, got `Url`" type errors.

Reply:
Yeah, I'm against ranges too

Reply:
> It's a bad idea, it makes it look like end users can continue to use 1.7, but cargo will happily bring url 2 in their project anyway and then you have "expected `Url`, got `Url`" type errors.

I did not know that.